shell/platform/android/test/io/flutter/embedding/android/FlutterActivityAndFragmentDelegateTest.java

```diff
@@ -59,12 +58,20 @@ public void setup() {
     // being tested.
     mockFlutterEngine = mockFlutterEngine();
 
-    // Create a fake Host, which is required by the delegate being tested.
-    fakeHost = new FakeHost();
-    fakeHost.flutterEngine = mockFlutterEngine;
-
-    // Create a spy around the FakeHost so that we can verify method invocations.
-    spyHost = spy(fakeHost);
+    // Create a mocked Host, which is required by the delegate being tested.
+    fakeHost = mock(FlutterActivityAndFragmentDelegate.Host.class);
```

Comment:
uber uber nit: you can call it mockHost at this point :D

Reply:
Done.